### PR TITLE
Document configmap section for remote-hosts

### PR DESCRIPTION
**Pull request by lkolacek**
Remove deprecated cluster section from documentation

CLOUDBLD-8995

Signed-off-by: Ladislav Kolacek <lkolacek@redhat.com>

### CODE REVIEW

**Review by hjmodi — APPROVED** [review]

File: docs/admins.rst [hunk 1]
```diff
@@ -71,29 +71,36 @@ token from the mounted secret file.
...
+  remote_hosts:
+    slots_dir: path/foo
+    pools:
+      x86_64:
+        osbs-remote-hosts-1-x86_64:
+          enabled: true
+          auth: /secret-path
+          username: podman-user
+          slots: 1
+          socket_path: /run/user/9999/podman-user/podman.sock
```
Comment (chmeliik) [inline docs/admins.rst:90]: might as well use the upstream default
```suggestion
          socket_path: /run/user/2022/podman/podman.sock
```

File: docs/admins.rst [hunk 2]
```diff
@@ -109,39 +116,35 @@ with content as in the example below::
...
-''''''''
+remote_hosts
+''''''''''''
 
-This maps each platform to a list of clusters and their concurrent
-build limits. For each platform to build for, a worker cluster is
-chosen as follows:
+This section provides a shared path to the slots directory.
+The slots directory holds files with information about currently built
+images.
```
Comment (chmeliik) [inline docs/admins.rst:124]:
```suggestion
This section provides a shared path to the slots directory.
The slots directory holds files with information about ongoing
builds.
```

**Review by chmeliik — APPROVED** [review]
LGTM :+1:

File: docs/admins.rst [hunk 3]
```diff
@@ -71,29 +71,36 @@ token from the mounted secret file.
...
-given time for each cluster. It will also list worker clusters in
-order of preference. It may also contain additional environment configuration
-such as ODCS integration.
+OSBS builds container images using podman-remote on remote VMs (hosts).
+OSBS needs only ssh key and hostname to connect to remote host and
+execute podman.
 
-The runtime configuration will take the form of a Kubernetes secret
-with content as in the example below::
+Remote host configuration example::
```
[the remark this answers is not in the document]
Comment (lkolacek) [reply docs/admins.rst]: Changed, please take a look

**Pull request merged**